### PR TITLE
Added scripts for the 2016-12-05-hoc-global-is-here mailing [ci skip]

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

File: bin/mail/2016-12-05-hoc-global-is-here/generate-contacts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    kind_s: K5ProfessionalDevelopmentSurvey ||
+    kind_s: ProfessionalDevelopmentWorkshop ||
+    kind_s: ProfessionalDevelopmentWorkshopSignup ||
+    kind_s: StudentNomination ||
+    kind_s: TeacherNomination
+  ',
+
+  non_teachers: '
+    kind_s: VolunteerEngineerSubmission2015 ||
+    kind_s: VolunteerEngineerSubmission2016 ||
```
Comment (ashercodeorg) [inline bin/mail/2016-12-05-hoc-global-is-here/generate-contacts]: This doesn't exist, AFAIK. The 2016 volunteers are included in the 2015 volunteers.

Comment (ashercodeorg) [reply bin/mail/2016-12-05-hoc-global-is-here/generate-contacts]: Do you want to include `VolunteerEngineerSubmission`?

Comment (aoby) [reply bin/mail/2016-12-05-hoc-global-is-here/generate-contacts]: Yeah you're right. I just confirmed in SOLR. There is only VolunteerEngineerSubmission and VolunteerEngineerSubmission2015. Confirming with Jon...

Comment (aoby) [reply bin/mail/2016-12-05-hoc-global-is-here/generate-contacts]: confirmed and updated. There are only 888 VolunteerEngineerSubmission entries in SOLR so it shouldn't effect the count significantly.

**Comment by ashercodeorg** [comment]
LGTM.

**Pull request merged**